lib/task.js

```diff
@@ -23,11 +23,16 @@ const getSymbol = task => {
 			return `${logSymbols.success} `;
 		case state.FAILED:
 			return task._result ? `${pointer} ` : `${logSymbols.error} `;
+		case state.SKIPPED:
+			return `${logSymbols.warning} `;
```

Comment:
Not sure if we should use warning for this state. Any suggestions for something else from [this](https://github.com/sindresorhus/figures#figures) list.